src/libexpr/flake/flakeref.hh

```diff
@@ -12,10 +12,25 @@ class Store;
 
 typedef std::string FlakeId;
 
+// The FlakeRef represents a local nix store reference to a flake
```

Comment:
The "local nix store reference" bit is confusing. Maybe something like "A flake reference specifies how to fetch a flake (e.g. from a Git repository). It is created from a URL-like syntax (e.g. 'github:NixOS/patchelf') or from an attrset representation (e.g. '{ type = "github"; owner = "NixOS"; repo = "patchelf"; }').".

Reply:
Thanks, that does sound clearer.  I made a couple of additional adjustments to this description that occurred to me as I was working on it; let me know what you think of the updated version.